openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -1256,7 +1256,7 @@
     "no-mentions": "There are no instances where you or your team have been referenced in any activities",
     "no-notification-found": "No notifications found",
     "no-open-task": "No Open Tasks",
-    "no-owned-data": "It appears that you or your team do not have any Data Assets under your ownership. To rectify this, kindly click on the \"Explore\" option to uncover and take ownership of available data assets.",
+    "no-owned-data": "It appears that you or your team do not have any Data Assets under your ownership. To rectify this, kindly click on the \"Explore\" on your menu to discover.",
```

Comment:
Can you update other language files as well

Reply:
done, Thanks for highlighting :)